### ISSUE TITLE
Dragon userbar compatibility

### ISSUE THREAD

**Issue by Stormheg** [wagtail/wagtail#10674]
### Wagtail userbar issue

The Wagtail userbar that appears at the bottom of live and draft pages is no longer usable with Dragon. I could previously use Dragon to activate this button in Wagtail 3. But it doesn't work in Wagtail 5 (I didn't try it when I tested Wagtail 4).

When I was viewing a live (or draft) page in Wagtail 3 I could say "click Wagtail" and this would bring up the userbar pop up menu. I could also say "click button" and then choose the number that corresponded to the Wagtail icon, just as I would do with anything else coded as a button. It doesn't respond to either of these commands in Wagtail 5.

If I use my mouse to activate the userbar, I can then use Dragon to select one of the options in the pop-up menu. For example, I can say "click edit this page" to open the visible page in the CMS. It's just the userbar itself that is no longer recognised by Dragon.

I'm using Dragon 15 and Firefox is my browser.

_Originally posted by @albfreeman in https://github.com/wagtail/wagtail/issues/9666#issuecomment-1621964468_

**Comment by Stormheg** [comment]
I'm investigating and will report more details here. I do not want to pollute the general tracking issue (9666) with my findings, hence I've created this issue.

**Comment by Stormheg** [comment]
I don't own a copy of Dragon, instead I checked if this issue also occurs for macOS Voice Control (shipped with macOS Ventura v13.4.1) + Google Chrome v114

Voice Control does not respond to the command 'click Wagtail' at all. I tested this in Wagtail v2.16, v4.0 and the current in-development v5.1 in [`main`](https://github.com/wagtail/wagtail/commit/c8823d62b726f4ecf3b912c6850fa2e5fc90c947)

Looking at the source code for the userbar, I can see the button is labeled as 'View wagtail quick actions' ([link](https://github.com/wagtail/wagtail/blob/c8823d62b726f4ecf3b912c6850fa2e5fc90c947/wagtail/admin/templates/wagtailadmin/userbar/base.html#L30)). Commanding 'click view wagtail quick actions' worked for all Wagtail versions.

**Comment by ankit02327** [comment]
Hi, I’d like to investigate this issue and test possible fixes around the userbar button labeling / speech recognition compatibility across assistive technologies.
